moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp

```diff
@@ -199,13 +199,13 @@ void FollowJointTrajectoryControllerHandle::controllerDoneCallback(
 {
   // Output custom error message for FollowJointTrajectoryResult if necessary
   if (!result)
-    ROS_WARN_STREAM_NAMED(LOGNAME, "Controller " << name_ << " done, no result returned");
+    ROS_WARN_STREAM_NAMED(LOGNAME, "Controller '" << name_ << "' done, no result returned");
   else if (result->error_code == control_msgs::FollowJointTrajectoryResult::SUCCESSFUL)
-    ROS_INFO_STREAM_NAMED(LOGNAME, "Controller " << name_ << " successfully finished");
+    ROS_INFO_STREAM_NAMED(LOGNAME, "Controller '" << name_ << "' successfully finished");
   else
-    ROS_WARN_STREAM_NAMED(LOGNAME, "Controller " << name_ << " failed with error "
-                                                 << errorCodeToMessage(result->error_code) << ": "
-                                                 << result->error_string);
+    ROS_WARN_STREAM_NAMED(LOGNAME, "Controller '" << name_ << "' failed with error "
+                                                  << errorCodeToMessage(result->error_code) << ": "
+                                                  << result->error_string);
```

Comment:
```suggestion
                                                  << result->error_string.empty() ? state.getText() : result->error_string);
```
maybe? maybe not?

Reply:
I already fixed it. The formatting job shows the suggested formatting. The `<<` needed to be moved to the right by 1 space.

Reply:
this wasn't about the formatting but about displaying the generic action error string in case the other one is empty

Reply:
`/friday_afternoon`.

Sorry.

Reply:
Should probably be added in a separate PR though.

Reply:
Hehe, let's not micro-PR things? It's the same topic after all

Reply:
> Hehe, let's not micro-PR things? It's the same topic after all

In theory I agree with @simonschmeisser, but I understand why we should not delay PRs because of other issues in the same code lines. Please create a quick follow-up PR for your improvement Simon! :medal_sports: